cf_remote/commands.py

```diff
@@ -285,7 +285,7 @@ def spawn(platform, count, role, group_name, provider=Providers.AWS, region=None
     key_pair = None
     if provider == Providers.AWS:
         try:
-            creds = AWSCredentials(creds_data["aws"]["key"], creds_data["aws"]["secret"])
+            creds = AWSCredentials(creds_data["aws"]["key"], creds_data["aws"]["secret"], creds_data["aws"]["token"])
```

Comment:
Token should be optional. This will cause a KeyError for me or anyone else who is using this without token.

Reply:
Ok. Will fix.